olp-cpp-sdk-authentication/include/olp/authentication/SignInUserResult.h

```diff
@@ -31,63 +31,66 @@ namespace authentication {
 class SignInUserResultImpl;
 
 /**
- * @brief The SignInUserResult class represents the response to a user sign-in
- * operation.
+ * @brief A response to your sign-in operation.
  *
- * If a HTTP status of 412 (PRECONDITION_FAILED) or 201 (CREATED) is returned,
- * the response will return terms of re-acceptance. This response may be
- * received on the first sign in, or any subsequent sign-in, for any user
- * authentication method, as long as the user has not accepted the latest terms
- * for their country.
+ * If the HTTP status of 412 (PRECONDITION_FAILED) or 201 (CREATED) is returned,
+ * the response returns terms of re-acceptance. Not to receive this response
```

Comment:
```suggestion
 * the response returns terms of re-acceptance. In order to not to receive this response
```
I don't know, your variant sounds a bit unusual for me (but maybe it is correct :) )

Reply:
"Not to" is a shorter version of "in order not to". Same as "to" is a shorer version of "in order to": *In order to/to* log in to the system, you have to provide your user credentials.